feature-group-definitions/colrv1.yml

```
@@ -1,4 +1,5 @@
 name: COLRv1
+description: The COLRv1 font format is used by color fonts to support multi-color glyphs.
```

For brevity:

```suggestion
description: COLRv1 is a font format that supports multi-color glyphs.
```

(That said, I'm a little nervous about using "supports" in these descriptions, but I couldn't think of an alternative here.)